src/common/pipe/snake-case-to-camel-case.pipe.ts

```diff
@@ -0,0 +1,39 @@
+import { PipeTransform } from '@nestjs/common';
+
+interface SnakeCaseToCamelCasePipeParameters {
+  prefixIgnorePattern?: string;
+  suffixIgnorePattern?: string;
+}
+
+export class SnakeCaseToCamelCasePipe
```

Comment:
Why not use https://docs.nestjs.com/techniques/serialization#transform

Reply:
may see https://github.com/typestack/class-transformer/issues/286

Reply:
"Stop Trying to Reinvent the Wheel"

Reply:
but I don't know how to use what you mentioned above to achieve the same effect singly using pipes